Vocable/Common/Views/EmptyStateView.swift

```diff
@@ -148,7 +148,7 @@ final class EmptyStateView: UIView {
             descriptionAttributedText = nil
         }
         button.setTitle(type.buttonTitle, for: .normal)
-        button.accessibilityIdentifier = "empty_state_addPhrase_button"
+        button.accessibilityIdentifier = AccessibilityID.shared.emptyStateAddPhraseButton.id
```

Comment:
Because this is a GazeableButton (as opposed to a config) we might be able to use the AccessibilityID property to simplify the identifier assignment, to something like: 
```suggestion
        button.accessibilityID =.shared.emptyStateAddPhraseButton
```